contrib/pg_tde/src/common/pg_tde_utils.c

```diff
@@ -91,7 +85,7 @@ get_all_tde_tables(void)
 	SysScanDesc scan;
 	HeapTuple	tuple;
 	List	   *tde_tables = NIL;
-	Oid			am_oid = get_tde_basic_table_am_oid();
+	Oid			am_oid = get_tde_table_am_oid();
```

Comment:
I would personally not have bothered to fix this dubious function but since the fix was tiny I do not care either way.